lib/jekyll-import/importers/dotclear.rb

```diff
@@ -1,121 +1,101 @@
+          file_path = opts["datafile"]
+          if file_path.nil? || file_path.empty?
+            Jekyll.logger.abort_with "Import Error:", "Dotclear export file not found!"
           end
 
-          elts = extract_data_section(line)
-
-          if type_data == "post"
-            draft = (elts[headers[type_data].index("post_status")] != "1")
-
-            date_str = elts[headers[type_data].index("post_creadt")]
-            date_blank = (date_str.nil? || date_str.empty?)
-            date_str_formatted = date_blank ? Date.today : Date.parse(date_str).strftime("%Y-%m-%d")
-            title_param = elts[headers[type_data].index("post_title")].to_s.parameterize
-
-            content = elts[headers[type_data].index("post_content_xhtml")].to_s
-            content = content.gsub('\"', '"').gsub('\n', "\n").gsub("/public/", "/assets/images/")
-
-            filepath = File.join(Dir.pwd, (draft ? "_drafts" : "_posts"), "#{date_str_formatted}-#{title_param}.html")
-
-            entire_content_file = <<~POST_FILE
-              ---
-              layout: post
-              title: "#{elts[headers[type_data].index("post_title")]}"
-              date: #{elts[headers[type_data].index("post_creadt")]} +0100
-              tags: ABC
-              ---
-
-              #{content}
-            POST_FILE
-
-            posts_and_drafts[elts[headers[type_data].index("post_id")]] = { :path => filepath, :content => entire_content_file }
-          elsif type_data == "media"
-            elts[headers[type_data].index("media_title")]
-            mediafilepath = elts[headers[type_data].index("media_file")]
-
-            src_path = File.join(options[:mediafolder], mediafilepath)
-            dst_path = File.join(Dir.pwd, "assets", "images", mediafilepath.to_s)
-
-            FileUtils.mkdir_p(File.dirname(dst_path))
-            FileUtils.cp(src_path, dst_path)
-          elsif type_data == "meta"
-            keywords[elts[headers[type_data].index("post_id")]] ||= []
-            keywords[elts[headers[type_data].index("post_id")]] << elts[headers[type_data].index("meta_id")]
-          elsif type_data == "link"
-
-          elsif type_data == "setting"
-
-          elsif type_data == "comment"
+          file_path = File.expand_path(file_path)
+          if File.open(file_path, "rb", &:readline).start_with?("///DOTCLEAR|")
+            @data = read_export(file_path)
+            Jekyll.logger.info "Export File:", file_path
+          else
+            Jekyll.logger.abort_with "Import Error:", "#{file_path.inspect} is not a valid Dotclear export file!"
+          end
+        end
+        # rubocop:enable Style/IfUnlessModifier
+
+        def process(_opts)
+          Jekyll.logger.info "Importing.."
+          tags = register_post_tags
+          posts = @data["post"]
+          FileUtils.mkdir_p("_drafts") unless posts.empty?
+
+          posts.each do |post|
+            date, title, content = post.values_at("post_creadt", "post_title", "post_content")
```

Comment:
`post_created`? Might have forgotten an e?

Reply:
The strings here mirror the header keys in the backup file (attached as part of this PR):
```
[post post_id,blog_id,user_id,cat_id,post_dt,post_tz,post_creadt,post_upddt,post_password,...]
```